fastify.js

```diff
@@ -304,7 +304,7 @@ function build (options) {
   }
 
   function hookIterator (fn, state, next) {
-    if (state.res.finished === true) return undefined
+    if (state.res.finished === true) return
```

Comment:
As far as I know, `return undefined` and `return` are the same thing.

Reply:
Yeah, and the rest of code uses `return` instead of `return undefined`

Reply:
They are the same, but there's a reason that this particular place uses `return undefined`. It's because the other `return` statement in the function returns a value, so it's better for this one to return an explicit value as well. Code is easier to follow if either all `return` statements return a value, or if none of them return a value. See ESLint's [`consistent-return`](https://eslint.org/docs/rules/consistent-return) rule for a bit more of an explanation.

Reply:
Done